### ISSUE TITLE
exposure time vs. distance

### ISSUE THREAD

**Issue by traut21** [corona-warn-app/cwa-app-android#1430]
From the FAQs I learned that the risk of exposure is calculated from distance and time.

What are the details how this is computed?

Is there a chart of time vs. distance or could someone provide the algorithm behind?

I wonder what the minimum contact time is,
e.g. when you pass someone in the underground train.

**Comment by codejus** [comment]
Here you can find more about the risk calculation and assessment in the documentation:
https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md#risk-score-calculation
and https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md

**Comment by traut21** [comment]
Yes, I can find 'more' there - but I do not find the details. How is this dB scale applied? 

I do see the 55 dB estimation (attenuation for 1.5 m) and a 63 db (for 3 m), but I do not see any dependency on time. Does this indicate that you need an exposure of at least 10 minutes at a close distance < 1.5 m?

**Comment by FMCumhaill** [comment]
@traut21 this might help you https://github.com/corona-warn-app/cwa-documentation/blob/master/transmission_risk.pdf

**Comment by traut21** [comment]
Thanks, that's plenty of stuff. Is this just the theory or is it actual how the app does work?

**Comment by traut21** [comment]
I've neither read nor fully understood the content of this description. But it reads as if it would leave the hard work to the ExposureNotification API. It might define some paramaters, but it does not name or explain the actual parameters in use.

Nothing about the essential factors of time, duration, distance or the aforementioned dB values

**Comment by daimpi** [comment]
@traut21 you're interested how the risk calculation works, right? We [recently walked through some examples](https://covidapps.slack.com/archives/C0194ML0MLN/p1602134945259900) in the community Slack which you can join [here](https://github.com/corona-warn-app/cwa-documentation/issues/379).

The config values which are currently live can be found [here](https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md#current-configuration) 🙂.
